src/components/event/textNote/TextNoteDisplay.tsx

```diff
@@ -137,9 +137,20 @@ const TextNoteDisplay: Component<TextNoteDisplayProps> = (props) => {
 
   const publishReactionMutation = createMutation({
     mutationKey: ['publishReaction', event().id],
-    mutationFn: commands.publishReaction.bind(commands),
-    onSuccess: () => {
-      console.log('succeeded to publish reaction');
+    mutationFn: (...params: Parameters<typeof commands.publishReaction>) =>
+      commands
+        .publishReaction(...params)
+        .then((promeses) => Promise.allSettled(promeses.map(timeout(5000)))),
+    onSuccess: (results) => {
+      const succeeded = results.filter((res) => res.status === 'fulfilled').length;
+      const failed = results.length - succeeded;
+      if (succeeded === results.length) {
+        console.log('succeeded to publish reaction');
+      } else if (succeeded > 0) {
+        console.log(`failed to publish reaction on ${failed} nodes`);
+      } else {
+        console.error('failed to publish reaction on all node');
+      }
```

Comment:
- 用語の統一のため、relaysに書き換えていただけると嬉しいです。
- 投稿失敗していることがわかるように `console.warn`にしていただけると嬉しいです。

```suggestion
      if (succeeded === results.length) {
        console.log('Succeeded to publish a reaction');
      } else if (succeeded > 0) {
        console.warn(`failed to publish a reaction on ${failed} relays`);
      } else {
        console.error('failed to publish reaction on all relays');
      }
```